tests/keras/layers/embeddings_test.py

```diff
@@ -4,8 +4,6 @@
 import keras.backend as K
 
 
-@pytest.mark.skipif((K.backend() == 'mxnet'),
-                    reason='MXNet backend does not support Embeddings yet.')
 @keras_test
```

Comment:
There are few RNN tests, that are skipped due to Embedding issue. Can you please enable those tests cases as well?

Reply:
enabled more test cases